### PR TITLE
fixes 8784: defines max_cpu_count for docker compute resource

### PR DESCRIPTION
**Pull request by witlessbird**

### CODE REVIEW

File: app/models/foreman_docker/docker.rb [hunk 1]
```diff
@@ -24,6 +24,10 @@ def max_memory
       16 * 1024 * 1024 * 1024
     end
 
+    def max_cpu_count
+      1
```
Comment (eLobato) [inline app/models/foreman_docker/docker.rb]: Why 1? ```Docker.info['NCPU']``` returns the number of CPUs

**Comment by witlessbird** [comment]
@eLobato: fixed; although I don't think this parameter makes much sense for Docker...

**Comment by eLobato** [comment]
Me neither, but just in case we need it in the future for the compute resource...

**Comment by eLobato** [comment]
Merged as dba2e433c0d40ca781e63e615c42552aac6804a7, thanks @witlessbird

**Pull request closed without merging**